### PR TITLE
chore: update README with new require usage

### PR DESCRIPTION
**Pull request by DominicKramer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,7 +72,13 @@ const Logger = winston.Logger;
...
 
 // Imports the Google Cloud client library for Winston
-const LoggingWinston = require('@google-cloud/logging-winston');
+
+// Node 6+
+const {LoggingWinston} = require('@google-cloud/logging-winston');
+
+// Node 4+
+// const LoggingWinston = require('@google-cloud/logging-winston').LoggingWinston;
+
```
Comment (ofrobots) [inline README.md]: While you're here, add a TS/ESM style important example as well?

File: README.md [hunk 2]
```diff
@@ -130,7 +136,7 @@ If you wish to set the LogEntry `trace` property with a custom value, then set a
 
 ```js
 const winston = require('winston');
-const transport = require('@google-cloud/logging-winston');
+const transport = require('@google-cloud/logging-winston').LoggingWinston;
```
Comment (ofrobots) [inline README.md]: I would favour an example optimizing for usability with Node 6+. I.e. destructuring assignment.

File: README.md (hunk 1 above)
Comment (DominicKramer) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (DominicKramer) [reply README.md]: Done

**Pull request merged**